portfolio/src/main/java/com/google/sps/servlets/DeleteResponseServlet.java

```diff
@@ -0,0 +1,26 @@
+package com.google.sps.servlets;
+
+import com.google.cloud.datastore.Datastore;
+import com.google.cloud.datastore.DatastoreOptions;
+import com.google.cloud.datastore.Key;
+import com.google.cloud.datastore.KeyFactory;
+import java.io.IOException;
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+
+/** Servlet responsible for deleting tasks. */
+@WebServlet("/delete-response")
+public class DeleteResponseServlet extends HttpServlet {
```

Comment:
This deserves special recognition! 👏 🏅 

Not only did you go above and beyond here on your own initiative, but you also did a great job doing it! I especially like how deleting messages doesn't cause. page refresh - it feels smooth and dynamic.